tests/rptest/scale_tests/many_clients_test.py

```diff
@@ -40,6 +40,7 @@ def __init__(self, *args, **kwargs):
             # Enable segment size jitter as this is a stress test and does not
             # rely on exact segment counts.
             'log_segment_size_jitter_percent': 5,
+            'target_quota_byte_rate': 31460000,  # 30MiB
```

Comment:
Can you add a comment here about why we are setting this?

Reply:
Will do